cli/onionshare_cli/web/share_mode.py

```diff
@@ -425,10 +425,7 @@ def render_logic(self, path=""):
                 # Render directory listing
                 filenames = []
                 for filename in os.listdir(filesystem_path):
-                    if os.path.isdir(os.path.join(filesystem_path, filename)):
-                        filenames.append(filename + "/")
-                    else:
```

Comment:
I am not sure why the trailing slashes were added in the first place since adding trailing space produces wrong link which always gives a 404 (at least not). And the logic was such that trailing slash where added only to directories inside a directory, and not root level. Not sure what changed, but this seems to fix the issue.